Godeps/Godeps.json

```diff
@@ -86,6 +86,10 @@
 			"Rev": "deed6eddef4f5e28c14d26d3c25b1fc454ce8baf"
 		},
 		{
+			"ImportPath": "github.com/davecgh/go-spew/spew",
+			"Rev": "2df174808ee097f90d259e432cc04442cf60be21"
```

Comment:
I think this is a transitive import? Want to try using https://github.com/davecgh/go-spew/releases/tag/v1.1.0 ?